themes/default/static/programs/aws-s3bucket-bucketpolicy-java/src/main/java/myproject/App.java

```diff
@@ -1,26 +1,29 @@
+package myproject;
+
 import com.pulumi.Pulumi;
-import com.pulumi.core.Output;
 import com.pulumi.aws.s3.Bucket;
 import com.pulumi.aws.s3.BucketPolicy;
-import com.pulumi.aws.s3.inputs.BucketPolicyPolicyArgs;
-import java.util.Map;
+import com.pulumi.aws.s3.BucketPolicyArgs;
+import static com.pulumi.codegen.internal.Serialization.*;
 
 public class App {
     public static void main(String[] args) {
         Pulumi.run(ctx -> {
             var bucket = new Bucket("myBucket");
 
-            var bucketArn = bucket.arn();
-            var policyDocument = bucketArn.apply(arn -> String.format("""
-                {
-                  "Version": "2012-10-17",
-                  "Statement": [{
-                    "Effect": "Allow",
-                    "Principal": {"Service": "lambda.amazonaws.com"},
-                    "Action": ["s3:PutObject", "s3:PutObjectAcl"],
-                    "Resource": "%s/*"
-                  }]
-                }""", arn));
+            var policyDocument = bucket.arn().applyValue(arn -> serializeJson(
```

Comment:
This was necessary because Java 11 doesn't support multiline strings specified in this way. We use `serializeJson` elsewhere for this, so I just went with that for consistency.